_topic_maps/_topic_map_rosa_hcp.yml

```diff
@@ -63,9 +63,47 @@ Distros: openshift-rosa-hcp
 Topics:
 - Name: Creating a cluster workshop
   Dir: creating_cluster_workshop
+  Distros: openshift-rosa-hcp
   Topics:
   - Name: Deploying a cluster
     File: cloud-experts-getting-started-hcp-for-hcp
+  - Name: Creating an admin user
+    File: cloud-experts-getting-started-admin
+  - Name: Setting up an identity provider
+    File: cloud-experts-getting-started-idp
+  - Name: Granting admin rights
+    File: cloud-experts-getting-started-admin-rights
+  - Name: Accessing your cluster
+    File: cloud-experts-getting-started-accessing
+---
+Name: Tutorials
+Dir: cloud_experts_tutorials
+Distros: openshift-rosa-hcp
+Topics:
+- Name: Tutorials overview
+  File: index
+- Name: ROSA with HCP activation and account linking
+  File: cloud-experts-rosa-hcp-activation-and-account-linking-tutorial
+- Name: ROSA with HCP private offer acceptance and sharing
+  File: cloud-experts-rosa-with-hcp-private-offer-acceptance-and-sharing
+- Name: Getting started with ROSA
+  Dir: cloud-experts-getting-started
+  Distros: openshift-rosa-hcp
+  Topics:
+  - Name: Creating an admin user
```

Comment:
This is already under the learning section. You can delete "Creating an admin user" from the Tutorials section.